modules/nw-ipfailover-configuration.adoc

```diff
@@ -0,0 +1,160 @@
+// Module included in the following assemblies:
+//
+// * networking/configuring-ipfailover.adoc
+
+[id="nw-ipfailover-configuration_{context}"]
+= Configuring IP failover
+
+As a cluster administrator, you can configure IP failover on an entire cluster, or on a subset of nodes, as defined by the label selector. You can also configure multiple IP failover deployment configurations in your cluster, where each one is independent of the others.
+
+The IP failover deployment configuration ensures that a failover pod runs on each of the nodes matching the constraints or the label used.
+
+This pod runs Keepalived which uses Virtual Router Redundancy Protocol (VRRP) among all the Keepalived daemons to ensure that the service on the watched port is available, and if it is not, Keepalived automatically floats the virtual IPs (VIP).
+
+For production use, make sure to set a `selector` with at least two nodes to select the nodes. Also, set a `replicas` value that matches the number of nodes for the given labeled `selector`.
+
+.Prerequisites
+
+* You are logged in to the cluster with a user with `cluster-admin` privileges.
+
+.Procedure
+
+* Create `Deployment` and `service` YAML files to configure IP failover:
++
+.Example `Deployment` YAML for IP failover configuration
+[source,yaml]
+----
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: ipfailover-keepalived <1>
+  labels:
+    ipfailover: kuard-keepalived
+spec:
+  strategy:
+    type: Recreate
+  replicas: 2
+  selector:
+    matchLabels:
+      ipfailover: kuard-keepalived
+  triggers:
+    type: ConfigChange
+  template:
+    metadata:
+      labels:
+        ipfailover: kuard-keepalived
+    spec:
+      serviceAccountName: ipfailover
+      privileged: true
+      hostNetwork: true
+      nodeSelector:
+        node-role.kubernetes.io/worker: ""
+      containers:
+      - name: openshift-ipfailover
+        image: quay.io/openshift/ose-keepalived-ipfailover
```

Comment:
We should provide example tag for this image, and tell user how to upgrade/update the ipfailover image when new image release and contains some fixes.